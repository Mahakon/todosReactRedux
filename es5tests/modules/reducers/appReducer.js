'use strict';

//import {combineReducers} from 'redux';
//import todosReducer from './todosReducer';
//import filterReduser from './filterReducer';

var Redux = require('redux');
var combineReducers = Redux.combineReducers;
var todosReducer = require('./todosReducer');
var filterReduser = require('./filterReducer');

var appReducer = combineReducers({
    todosArray: todosReducer,
    currentFilter: filterReduser
});

module.exports = appReducer;