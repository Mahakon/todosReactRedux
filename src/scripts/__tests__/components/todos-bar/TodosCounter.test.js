import { mount, shallow } from 'enzyme'
import { compose } from 'redux'
import toJson from 'enzyme-to-json'
import TodosCounterUI from '../../../componets/ui/todos-bar/TodosCounterUI'
import storeFactory from '../../../store/storeFactory'
import appReducer from '../../../modules/reducers/appReducer'
import { Provider } from 'react-redux'

describe('<TodosCounter/> snap-shot test', () => {
   const shallowExpect = compose(expect, toJson, shallow);
    let store;
    let wrapper;
    beforeAll(() => {
        //TODO: just change 1 to 0
        store = storeFactory(testStates[1], appReducer)
    });

   it ('Renders correct props', () => {
        shallowExpect(
            <TodosCounterUI todosArray={store.getState().todosArray}/>
        ).toMatchSnapshot()
   })
});