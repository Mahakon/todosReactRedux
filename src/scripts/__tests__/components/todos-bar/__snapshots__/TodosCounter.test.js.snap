// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TodosCounter/> snap-shot test Renders correct props 1`] = `
<div
  aria-label="number of undone items"
  className="todos-actions-bar_counter-undone"
>
  2 items left
</div>
`;
